ss/pebbledb/db.go

```diff
@@ -482,6 +505,25 @@ func prependStoreKey(storeKey string, key []byte) []byte {
 	return append(storePrefix(storeKey), key...)
 }
 
+// Parses store from key with format "s/k:{store}/..."
+func parseStoreKey(key []byte) string {
```

Comment:
If this is very frequently accessed, one can help performance by avoiding some of the splits
```
const prefix = "s/k:"

func parseStoreKey(key []byte) string {
	// Convert byte slice to string only once
	keyStr := string(key)

	if !strings.HasPrefix(keyStr, prefix) {
		return ""
	}

	// Find the first occurrence of "/" after the prefix
	// This avoids splitting the entire string unnecessarily
	slashIndex := strings.Index(keyStr[len(prefix):], "/")
	if slashIndex == -1 {
		return ""
	}

	// Return the substring between the prefix and the first "/"
	return keyStr[len(prefix) : len(prefix)+slashIndex]
}
```

Reply:
Will update, this is cleaner. However this shouldn't be run too often (its basically only run once per module).

Reply:
Updated